tools/SetupFlow/DevHome.SetupFlow/Views/SetupTargetView.xaml

```diff
@@ -374,5 +377,16 @@
                 </StackPanel>
             </StackPanel>
         </Grid>
+        <setupControls:SetupShell.SetupShellNotification>
+            <InfoBar

```

Comment:
Probably depends on if we're going to add more of these, but should we create a Style for these InfoBars?

Reply:
Not sure, but I guess if other features start using it then it would be good to make a style in the future.